Palindrome/script.js

```diff
@@ -0,0 +1,20 @@
+function palindrome(str) {
+
+  let count = 0;
+  let arr = str.toLowerCase().split('').filter(char => /[a-z]/.test(char));
+  // console.log(arr);
+  let halfLength = Math.floor(arr.length/2); 
+
+  for (let i = 0; i < halfLength; i++) {
+
+    if (arr[i] == arr[arr.length - 1 - i]) {
+      count++;
+    }
+  }
+
+  if (count == halfLength) return true
```

Comment:
add curly braces so the code will be more readable and understandable
if() {
    ....
} else {
    ....
}

You can use: if() return value  - if you have one row only

Reply:
I now it , but sometimes i so do it , as the code is very short.
ok, I`ll improve

Reply:
know😁